test/AutoRest.TestServer.Tests/Infrastructure/TestServerSession.cs

```diff
@@ -104,7 +104,7 @@ public async ValueTask DisposeAsync(bool ignoreChecks)
                 {
                     foreach (var expectedStub in _expectedCoverage)
                     {
-                        if (!matched.Contains(expectedStub, StringComparer.InvariantCultureIgnoreCase))
+                        if (matched.Length > 0 && !matched.Contains(expectedStub, StringComparer.InvariantCultureIgnoreCase))
```

Comment:
Why do we need this?

Reply:
For [these tests](https://github.com/Azure/autorest.csharp/blob/ecf8399bb59fbf9b7df1ba34988440febf7752a4/test/AutoRest.TestServer.Tests/body-complex.cs#L63) it is going inside of this condition and giving this error:
![image](https://user-images.githubusercontent.com/45216704/89474359-370e9900-d73a-11ea-81f4-e8f8fbfab9a5.png)

and the length of `matched` is 0, that's why I added this check.

Reply:
When I was using `Test((host, pipeline) =>` for [these tests](https://github.com/Azure/autorest.csharp/blob/ecf8399bb59fbf9b7df1ba34988440febf7752a4/test/AutoRest.TestServer.Tests/body-complex.cs#L63), it was going inside of `if` condition and giving this error:
![image](https://user-images.githubusercontent.com/45216704/89474359-370e9900-d73a-11ea-81f4-e8f8fbfab9a5.png)

because the length of `matched` was 0 for these cases, that's why I added this check but now that I've removed `Test((host, pipeline)`, this condition not needed.